bladex/propeller.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module for the propeller with shaft bottom-up parametrized construction.
+"""
+import os
+import numpy as np
+from bladex import Blade, Shaft
+import OCC.Core.TopoDS
+from OCC.Core.gp import gp_Dir, gp_Pnt, gp_Ax1, gp_Trsf
+from OCC.Core.IGESControl import IGESControl_Reader, IGESControl_Writer
+from OCC.Core.BRepBuilderAPI import BRepBuilderAPI_Transform, BRepBuilderAPI_Sewing
+from OCC.Core.BRepAlgoAPI import BRepAlgoAPI_Fuse
+from OCC.Extend.DataExchange import write_stl_file
+
+class Propeller(object):
+    """
+    Bottom-up parametrized propeller (including shaft) construction.
+
+    :param shaft.Shaft shaft: shaft to be added to the propeller
+    :param blade.Blade blade: blade of the propeller
+    :param int number_of blades: number of blades componing the propeller
+    """
+
+    def __init__(self, shaft, blade, number_of_blades):
+        self.shaft = shaft
+        self.blade = blade
+        self.number_of_blades = number_of_blades
+
+    def generate_propeller(self, propeller_and_shaft):
+        """
+        Generate and export the .iges and .stl CAD for the propeller with shaft.
+        This method requires PythonOCC (7.4.0) to be installed.
```

Comment:
I would remove the version dependency from these docstrings: all the package needs 7.4.0 right?

Reply:
That's fine. Thanks.